src/style.rs

```diff
@@ -235,10 +249,9 @@ impl ProgressStyle {
                     alt_style,
                 } => {
                     buf.clear();
-                    if let Some(formatter) = self.format_map.get(key.as_str()) {
-                        buf.push_str(&formatter(state));
-                    } else {
-                        match key.as_str() {
+                    match self.format_map.get(key.as_str()) {
```

Comment:
I'd prefer to minimize the changes here to make this easier to review.

Reply:
Ah yeah that was leftover from the previous way I was doing it, I agree it's more idiomatic the previous way now